schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -537,43 +533,32 @@
             }
         },
         "containers": {
+            "$comment": "TODO: description needed",
```

Comment:
A set of structures (called containers) used to store vulnerability information related to a specific CVE ID.  Each container includes information provided by a different source.  At minimum, a cnaContainer containing the vulnerability information provided by the CNA who initially assigned the CVE ID must be included.

There can only be one cnaContainer, as there can only be one assigning CNA.  However, there can be multiple adpContainers.  For the most part, the cnaContainer and adpContainers contain the same fields.  The main differences are the source of the information and the cnaContainer requires the CNA include certain fields.